data/autoyast_sle15/set_passwords.sh.1

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
s/.sh.1/sh/

Reply:
Good catch, the file already existed, so wget added that .1. I verified, it is strictly the same file, so I removed the duplicate.